### PR TITLE
Update CONTRIBUTING.md for mentioning accepted languages and meaningf…

### LINKED ISSUE #216: Update CONTRIBUTING.md
CONTRIBUTING.md needs some updates:
1. Mention of currently accepted languages.
2. Mention use of meaningful function/method names.

### PR DESCRIPTION
**Pull request by Monal5031**
…ul function/method names.

Fixes #216

#### By submitting this pull request I confirm I've read and complied with the below declarations.

- [x] I have read the [Contribution guidelines](https://github.com/iiitv/algos/blob/master/CONTRIBUTING.md) and I am confident that my PR reflects them.
- [ ] I have followed the [coding guidelines](https://github.com/iiitv/algos/blob/master/CONTRIBUTING.md#cs) for this project.
- [ ] My code follows the [skeleton code structure](https://github.com/iiitv/algos/blob/master/CONTRIBUTING.md#sample).
- [x] This pull request has a descriptive title. For example, `Added {Algorithm/DS name} [{Language}]`, not `Update README.md` or `Added new code`.
- [x] This pull request will be closed if I fail to update it even once in a continuous timespan of 7 days.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -32,8 +32,9 @@
 * Filename should be derived from the folder name. (eg `longest_common_subsequence` becomes `longestCommonSubsequence.c` or `LongestCommonSubsequence.java`)
 * Name of master function of the code should be kept same as filename to the best extent possible.
 * Prefer classes instead of multiple helper functions (where applicable).
+* Currently we are accepting contributions in C, C++, Java, Python, GO but other languages may be considered after a discussion.
```
Comment (singhpratyush) [inline CONTRIBUTING.md]: `... and go ...`

Comment (aviaryan) [inline CONTRIBUTING.md]: `Go` with a captial 'g'. https://golang.org/

**Comment by aashutoshrathi** [comment]
@Monal5031 while stating

> Filename should be derived from the folder name. (eg `longest_common_subsequence` becomes `longestCommonSubsequence.c` or `LongestCommonSubsequence.java`)

Should we now mention that one can use `lowerCamelCase` or `under_scores` for file name in Python, C and C++ ?
Views @aviaryan @singhpratyush ?

**Comment by singhpratyush** [comment]
> Should we now mention that one can use `lowerCamelCase` or `under_scores` for file name in Python, C and C++ ?

That is not needed to mention. Also, only `under_scores` for Python.

**Comment by singhpratyush** [comment]
> Should we now mention that one can use `lowerCamelCase` or `under_scores` for file name in Python, C and C++ ?

That is not needed to mention. Also, only `under_scores` for Python.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -32,8 +32,9 @@
 * Filename should be derived from the folder name. (eg `longest_common_subsequence` becomes `longestCommonSubsequence.c` or `LongestCommonSubsequence.java`)
 * Name of master function of the code should be kept same as filename to the best extent possible.
 * Prefer classes instead of multiple helper functions (where applicable).
+* Currently we are accepting contributions in C, C++, Java, Python and Go but other languages may be considered after a discussion.
 * Define `tester` code only in `main` routine.
-* Use meaningful variable names and comments.
+* Use meaningful variable, methods/function names and comments.
```
Comment (aviaryan) [inline CONTRIBUTING.md]: I think "variable, method and function names" would be better. 

Notice I changed methods to method

**Pull request merged**